app/public/javascripts/vendor/jquery.growl.js

```diff
@@ -0,0 +1,272 @@
+/* eslint-disable */
+
+/*
+jQuery Growl
```

Comment:
Why you adding this here and not just setting it as public from `app.js` file?

Reply:
Something like (using the corresponding file):
```node
  app.use('/jquery', express.static(`${__dirname}/node_modules/jquery/dist/`));
```

Reply:
😮

Reply:
What about this?